src/SonarQube.TeamBuild.Integration/SonarQube.TeamBuild.Integration.csproj

```diff
@@ -9,12 +9,7 @@
     <ProjectReference Include="..\SonarQube.Common\SonarQube.Common.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Services.Client" version="14.102.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Services.InteractiveClient" version="14.102.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" version="1.14.114" />
-    <PackageReference Include="Microsoft.WindowsAzure.ConfigurationManager" version="1.7.0.0" />
-    <PackageReference Include="Newtonsoft.Json" version="10.0.3" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" version="4.0.2.206221351" />
-    <PackageReference Include="WindowsAzure.ServiceBus" version="3.3.2" />
+    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.14.114" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.4.0" />
```

Comment:
`Microsoft.Win32.Registry` on a cross-platform application looks weirdo to me but...

Reply:
Yeah, that's odd but apparently it is cross platform.

Reply:
Looks weird to me too.